docs/dashboards-new/dashboards-migration/index.md

```diff
@@ -0,0 +1,23 @@
+---
+id: index
```

Comment:
needs to be a slug (refer to any index page to see how to do it)

Reply:
Updated accordingly.